src/main/java/org/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,37 @@
+package org.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.antlr.v4.runtime.misc.NotNull;
+
+@Entity
+@Data
```

Comment:
```suggestion
@Getter
@Setter
@Table(name....
```

don't use `@Data` for entities